.travis.yml

```diff
@@ -7,14 +7,16 @@ python:
   - "3.6"
   - "3.7"
 
+addons:
```

Comment:
What are the advantages of installing dependencies like this?

Reply:
Less manual steps and iirc an implied `apt-get update`